Sources/ArcGISToolkit/Components/AR/TableTopSceneView.swift

```diff
@@ -170,12 +178,15 @@ public struct TableTopSceneView: View {
               let planeGeometry = ARSCNPlaneGeometry(device: device)
         else { return }
         
+        // Disable coaching overlay when a plane node is found.
+        coachingOverlayIsActive = false
```

Comment:
does this ever need to be set to true again? Can nodes/planes go away?

Reply:
No, the nodes are only removed with `node.removeFromParentNode()` when the ArcGIS Scene is added to the view, so the coaching overlay is not needed at that point. If we [support](https://developer.apple.com/documentation/arkit/arsessionobserver/2941046-sessionshouldattemptrelocalizati) relocalization then the coaching overlay may need to be shown again.